pkg/log/accesslog.go

```diff
@@ -152,11 +153,14 @@ func parseFormat(format string) ([]*logEntry, error) {
 					}
 
 					// var def ends, add variable
-					_, err := variable.AddVariable(format[lastMark+1 : pos])
+					varName := format[lastMark+1 : pos]
+					_, err := variable.AddVariable(varName)
 					if err != nil {
-						return nil, err
+						// adapte istio unknow fields
```

Comment:
The format support this style `"log_format": "%StartTime% %RequestReceivedDuration% %ResponseReceivedDuration% %REQ.requestid% %REQ.cmdcode% %RESP.requestid% %RESP.service%"`.